app/frontend/src/utils/arrayEq.ts

```diff
@@ -0,0 +1,10 @@
+export function arrayEq(a: any[], b: any[]) {
```

Comment:
```suggestion
export function arrayEq(a: (number | string | boolean)[], b: (number | string | boolean)[]) {
```
I think we probably want to restrict the list to only contain primitive values so we don't have to deal with cases of values in `a` or `b` being an object?